### PR TITLE
minor tweak of finite difference atomic_displacements order

### PR DESCRIPTION
**Pull request by psi-rking**
## Description
Minor tweaks to finite differences in the atomic_displacements() function that is used only in prototype ROA calculations at present.  

## Todos
Now supports 3-point and 5-point formulae finite differences in the atomic_displacements() function.
Now returns displacements in canonical most negative to most positive order.

* **Developer Interest**
I have used this function in dipole moment derivative/file17 computations.  Further development along these lines is likely.

* **User-Facing for Release Notes**
None at this juncture.

## Questions
None

## Status
Should be ready to go.

### CODE REVIEW

File: psi4/src/psi4/findif/fd_misc.cc [hunk 1]
```diff
@@ -270,36 +270,52 @@ void displace_atom(SharedMatrix geom, const int atom, const int coord, const int
...
+        SharedMatrix m1_geom(ref_geom->clone());
+        displace_atom(m1_geom, atom, coord, -1, disp_size);
+        disp_geoms.push_back(m1_geom);
+        // plus displacements
+        SharedMatrix p1_geom(ref_geom->clone());
+        displace_atom(p1_geom, atom, coord, +1, disp_size);
+        disp_geoms.push_back(p1_geom);
+        SharedMatrix p2_geom(ref_geom->clone());
+        displace_atom(p2_geom, atom, coord, +2, disp_size);
+        disp_geoms.push_back(p2_geom);
```
Comment (loriab) [inline psi4/src/psi4/findif/fd_misc.cc]: Could just wrap `if (pts == 5)` around the "p2"/"m2" blocks and condense. But not vital.

Comment (dgasmith) [inline psi4/src/psi4/findif/fd_misc.cc]: This would be cleaner, would be good to put an except in the `else` statement as well.

**Comment by psi-rking** [comment]
I rather prefer the readability of separate blocks, particularly if other options might be supported in the future, but I'll change if you want.  Just added the exception (probably too used to relying on our options class for that).

**Comment by psi-rking** [comment]
I just discovered I had to switch over to the new "properties()" keyword, but it looks great!

**Comment by dgasmith** [comment]
@psi-rking I think is ready to go, all set?

**Comment by psi-rking** [comment]
Yes, it's ready to go.

**Pull request merged**